index.js

```diff
@@ -1,6 +1,7 @@
 var proxy = require('http-proxy-middleware');
 
 module.exports = (api, options) => {
+	if (options.pluginOptions === undefined) return;
 	const opts = options.pluginOptions.proxy;
```

Comment:
Account for `null` as well?
`const opts = options.pluginOptions && options.pluginOptions.proxy;`